lib/Froxlor/Api/Commands/DomainZones.php

```diff
@@ -138,45 +138,51 @@ public function add()
 			$errors[] = $this->lng['error']['dns_arec_noipv4'];
 		} elseif ($type == 'AAAA' && filter_var($content, FILTER_VALIDATE_IP, FILTER_FLAG_IPV6) === false) {
 			$errors[] = $this->lng['error']['dns_aaaarec_noipv6'];
-		} elseif ($type == 'MX') {
-			if ($prio === null || $prio < 0) {
-				$errors[] = $this->lng['error']['dns_mx_prioempty'];
-			}
+		} elseif ($type == 'CAA' && ! empty($content)) {
+			// check that CAA content is enclosed in " "
+			$content = \Froxlor\Dns\Dns::encloseTXTContent($content);
```

Comment:
only requirement is enclosure in " "?
I'd preg_split it, regex to see it the first part is `0` and the second part one of `issue`, `issuewild` and `iodef` and the third a valid domain

Reply:
what about this? (just posting this so I can check `request changes` again :p

Reply:
What do you think about bfb3fb0a92bb02308797661eb549fc34d3711122?